README.md

```diff
@@ -121,3 +121,34 @@ warnings and will fail on execution because of port type mismatches.
 
 [Here](port-matching/workflow-value-from.cwl) is the same workflow with
 `valueFrom` added to make the port types match.
+
+
+## Some commonly used Javascript expression tools
+You can tailor the input/output types to your situation.
+
+### Flatten a nested array into an array
+[This](javascript_snippets/flatten-nestedarray.cwl) tool flattens an
```

Comment:
```suggestion
[flatten-nestedarray.cwl](javascript_snippets/flatten-nestedarray.cwl) shows how to flatten an
```
I find using the name of the file in the text to be more readable in the rendered version. Can you switch to this style everywhere?